src/js/utils.js

```diff
@@ -47,15 +47,50 @@ const changeAnchorLink = (anchorElement, { user, repository, typeOfSearch, date
   }
 }
 
-export const validateForm = (formData, anchorElement) => {
-  const filterData = createSearchFilter(formData);
+export const validateForm = () => {
+  const fielValid = {
+    type_of_search: {
+      required: true,
+    },
+    input_user: {
+      required: true,
+    },
+    input_repository: {
+      required: true,
+    },
+    input_condition: {
+      required: true,
+    },
+    input_date: {
+      required: true,
+    },
+  };
+  const fiels = document.querySelectorAll(
+    ".box.d-inline input,.box.d-inline select"
+  );
 
-  const isValid = filterData.user && filterData.repository && filterData.typeOfSearch && filterData.date;
+  let bool = true;
+  for (var i in fiels) {
+    if (!fiels.hasOwnProperty(i)) continue;
```

Comment:
Please consider to use for Each instead of For, with forEach you doesn't need to check the "hasOwnProperty", and you can do "return" instead of continue

```suggestion
  fields.forEach((field) => {
    // if (!fiels.hasOwnProperty(i)) return; // You can use "return" instead of continue, but it is not needed since forEach doesn't return that properties that you don't need, then you can remove this line of code 
  /// ... rest of the code
  ```